packages/react-native-reanimated/src/layoutReanimation/web/componentUtils.ts

```diff
@@ -148,10 +148,12 @@ export function setElementAnimation(
 ) {
   const { animationName, duration, delay, easing } = animationConfig;
 
-  element.style.animationName = animationName;
-  element.style.animationDuration = `${duration}s`;
-  element.style.animationDelay = `${delay}s`;
-  element.style.animationTimingFunction = easing;
+  requestAnimationFrame(() => {
```

Comment:
Have you checked if it works with `setTimeout(() => {}, 0)` instead of `requestAnimationFrame`?

Reply:
I've just tested that - it seems to work (though I think I've seen one flicker, but I couldn't reproduce it afterwards)